src/components/extensions/Headings.js

```diff
@@ -127,37 +123,61 @@ export const Warning = styled.div`
   background-color: #ffd000;
 `;
 
-const Image = ({ to }) => {
+const BannerImage = ({ to }) => {
   switch (to) {
     case "/messaging/":
-      return <StyledImage src={imgMessaging} />;
+      return (
+        <Image
+          src="/images/livechat-platform-messaging.png"
+          alt="Livechat platform messaging"
+          layout="responsive"
+          width={782}
+          height={682}
+        />
+      );
     case "/extending-ui/":
-      return <StyledImage src={imgExtendUI} />;
+      return (
+        <Image
+          src="/images/livechat-platform-extend-interfaces.png"
+          alt="Livechat platform extend interfaces"
+          layout="responsive"
+          width={782}
+          height={682}
+        />
+      );
     case "/data-reporting/":
-      return <StyledImage src={imgDataReporting} />;
+      return (
+        <Image
+          src="/images/livechat-platform-data-reporting.png"
+          alt="Livechat platform data reporting"
+          layout="responsive"
+          width={782}
+          height={682}
+        />
+      );
     default:
       return null;
   }
 };
 
-// const SectionBanner = ({ title, desc, to, image }) => (
-//   <SectionBannerWrapper>
-//     {image && (
-//       <SectionColumn style={{ padding: 0 }}>
-//         <Link to={to} style={{ display: "block" }}>
-//           <Image to={to} />
-//         </Link>
-//       </SectionColumn>
-//     )}
-//     <SectionColumn>
-//       <h2>{title}</h2>
-//       <p>{desc}</p>
-//       <Link to={to}>
-//         <Button>Learn more</Button>
-//       </Link>
-//     </SectionColumn>
-//   </SectionBannerWrapper>
-// );
+const SectionBanner = ({ title, desc, to, image }) => (
+  <SectionBannerWrapper>
+    {image && (
+      <SectionColumn style={{ padding: 0 }}>
+        <Link href={to} style={{ display: "block" }}>
```

Comment:
It's missing `<a>` tag here. Check [documentation](https://nextjs.org/docs/migrating/from-gatsby#links)